pkgs/top-level/linux-kernels.nix

```diff
@@ -181,12 +181,17 @@ in {
       ];
     };
 
-    linux_testing = callPackage ../os-specific/linux/kernel/linux-testing.nix {
-      kernelPatches = [
-        kernelPatches.bridge_stp_helper
-        kernelPatches.request_key_helper
-      ];
-    };
+    linux_testing = let
+      latest = packageAliases.linux_latest.kernel;
+      testing = callPackage ../os-specific/linux/kernel/linux-testing.nix {
+        kernelPatches = [
+          kernelPatches.bridge_stp_helper
+          kernelPatches.request_key_helper
+        ];
+      };
```

Comment:
It's a small thing, but could we switch the order of the declarations of `latest` and `testing`? It's a bit confusing to see the definition for `linux_testing` _start_ by getting the latest kernel, rather than starting with the testing kernel.